sinks/dashboard/items/plot_dash_item.py

```diff
@@ -162,6 +175,13 @@ def on_data_update(self, stream, payload):
                            if self.points[item] else 0 for item in self.series]
             for v in last_values:
                 current_values += f"[{v: < 4.4f}] "
+        if self.parameters.param('Show Slope').value():
+            slope_values = []
+            for stream in self.series:
+                slope = self.calculate_slope(self.times[stream], self.points[stream],
+                                            self.parameters.param('Number of Points').value())
+                slope_values.append(f"[{slope:.2f}]")
+            current_values += f"    Slope: {' '.join(slope_values)}"
```

Comment:
Specify that this is per second (and confirm that it is actually per second)
```suggestion
            current_values += f"    Slope (/sec): {' '.join(slope_values)}"
```